src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -454,11 +454,16 @@ private function buildTableRows($rows)
      * @param int   $line
      *
      * @return array
+     *
+     * @throws InvalidArgumentException
      */
     private function fillNextRows(array $rows, $line)
     {
         $unmergedRows = array();
         foreach ($rows[$line] as $column => $cell) {
+            if (null !== $cell && !$cell instanceof TableCell && !is_scalar($cell) && !(is_object($cell) && method_exists($cell, '__toString'))) {
+                throw new InvalidArgumentException(sprintf('The cell should always be a string, a TableCell or a scalar, %s given.', gettype($cell)));
```

Comment:
I would say `A cell must be a TableCell, a scalar or an object implementing __toString, %s given.`

Reply:
ping @Simperfit

Reply:
done